.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "compiler-wasm/static/build/packages/gleam_stdlib"]
+	path = compiler-wasm/static/build/packages/gleam_stdlib
+	url = git@github.com:gleam-lang/stdlib.git
```

Comment:
In adding the stdlib as a dependency of the compiler we've now got a circular dependency cycle. The stdlib depends on the compiler, and the compiler depends upon the stdlib.
I don't think this is the ideal design, and it means we're no longer able to do breaking language changes as there's no stdlib version that will support the new compiler behaviour.

We would also need to update the release build scripts and the container image build scripts.

I think the best option is to have the wasm based compiler work the same as the build tools and have the stdlib be passed in as normal Gleam code.

Reply:
Done :+1:

Reply:
@JohnDoneth So the git submodule and test-musl changes can be reverted now? Or is that still used?

Reply:
Yes, thanks, I will revert those changes

Reply:
Cool! I think it would be good to merge in main or rebase the brach as well since there's a conflict showing up (at least right now for me)

Reply:
Is this still needed?

Reply:
Removed